packages/cli/src/runners/task-managers/data-request-response-stripper.ts

```diff
@@ -0,0 +1,131 @@
+import type { DataRequestResponse, BrokerMessage } from '@n8n/task-runner';
+import type {
+	EnvProviderState,
+	IPinData,
+	IRunData,
+	IRunExecutionData,
+	ITaskDataConnections,
+} from 'n8n-workflow';
+
+/**
+ * Strips data from data request response based on the specified parameters
+ */
+export class DataRequestResponseStripper {
```

Comment:
This have been extracted from `DataRequestResponseBuilder`. No new functionality here